packages/web3/package.json

```diff
@@ -86,21 +86,21 @@
 		"web3-providers-ipc": "^4.0.7"
 	},
 	"dependencies": {
-		"web3-core": "^4.3.2",
-		"web3-errors": "^1.1.4",
-		"web3-eth": "^4.6.0",
-		"web3-eth-abi": "^4.2.1",
+		"web3-core": "^4.4.0",
+		"web3-errors": "^1.2.0",
+		"web3-eth": "^4.7.0",
+		"web3-eth-abi": "^4.3.0",
 		"web3-eth-accounts": "^4.1.2",
-		"web3-eth-contract": "^4.4.0",
-		"web3-eth-ens": "^4.2.0",
+		"web3-eth-contract": "^4.5.0",
+		"web3-eth-ens": "^4.3.0",
 		"web3-eth-iban": "^4.0.7",
 		"web3-eth-personal": "^4.0.8",
-		"web3-net": "^4.0.7",
+		"web3-net": "^4.1.0",
 		"web3-providers-http": "^4.1.0",
 		"web3-providers-ws": "^4.0.7",
-		"web3-rpc-methods": "^1.2.0",
+		"web3-rpc-methods": "^1.3.0",
 		"web3-types": "^1.6.0",
```

Comment:
there is types changelog update but this dep version is not updated

Reply:
https://github.com/web3/web3.js/blob/4.x/packages/web3-types/package.json
web3-types is currently 1.6.0, only the changelog was not updated